otx/cli/utils/report.py

```diff
@@ -116,6 +116,12 @@ def task_config_to_str(task_config: Dict[str, Any]):
     report_str = ""
     not_target = ["log_config"]
     for target, value in task_config.items():
+        # Remove otx_dataset
```

Comment:
```suggestion
        # Remove otx_dataset to avoid too long report
```
It couldn't be matter but how about adding an intention for removing otx_dataset?

Reply:
I corrected this to `# Remove otx_dataset from the report as it is unnecessary.`. Thanks for the comment.